dev/tasks/utils/cs-build-entry.js

```diff
@@ -0,0 +1,16 @@
+// jshint ignore: start
```

Comment:
This is more of an asset rather than utility. In a sense that it's not bringing value for the grunt task itself, but is used as src for bundling.

I feel like that file should in the end go to the `easyimage` directory. How about `plugins/easyimage/src/buildentry.js`? This directory should be removed by the builder (though `plugins/easyimage/lib/cs.js` should not ofc).

Any thoughts on it @Comandeer @f1ames?

Reply:
It is part of a Webpack task, but indeed `easyimage` specific. Moving to `plugins/easyimage` makes sense.

Couldn't we go a step further and make `webpack` task generic so if the plugin has a build script e.g. `plugins/easyimage/src/build-cs.js` it runs it and outputs the resulted file to some build dir `plugins/easyimage/lib/cs.js`? (well, I'm not sure how hard it is to do with Grunt...)

Reply:
> Couldn't we go a step further and make webpack task generic

I don't think we need it at this point. As soon as we face a case that we have another solution using webpack task, we'll generalize it.